app/submitter/convert_payload_0_0_1.py

```diff
@@ -73,48 +82,55 @@ def convert_answers_to_payload_0_0_1(
     return data
 
 
-def _format_downstream_answer(answer_type, answer_value, answer_data):
-    if answer_type == "Date":
-        return (
-            datetime.strptime(answer_value, "%Y-%m-%d")
-            .replace(tzinfo=timezone.utc)
-            .strftime("%d/%m/%Y")
-        )
+def _format_downstream_answer(
+    answer_type: str, answer_value: AnswerValueTypes, answer_data: str
+) -> str:
+    if isinstance(answer_value, str):
```

Comment:
This is interesting as I don't think Date and MonthYearDate answer_value can be anything but string however due to the generic nature of _format_downstream_answer and type hinting I can see why this has to be done

Reply:
Some of these need some serious refactoring..